src/utils/config.ts

```diff
@@ -70,6 +70,7 @@ export const getProjectChainInfo = (): ChainInfo => {
     viemChain: {
       instance: viemInstance,
     },
+    isTestnet: true,
```

Comment:
Since it's a flag that describe that this chain is a testnet or not, I don't think we should hardcode in this case because this flag could be used by other case and would make the result become incorrect.